README.md

```diff
@@ -68,3 +102,32 @@ Most formatters have some options available that can be configured via Glaive.
 You can get a quick view of all codefmt flags by executing `:Glaive codefmt`, or
 start typing flag names and use tab completion. See `:help Glaive` for usage
 details.
+
+## Creating a New Formatter
```

Comment:
Added a link. 

I think there's value in having a quick guide here, esp. since the README's not that long yet.

Long-term, I think the best solution is the combination of a Guide + a well-commented Example file.